rmf_traffic/src/rmf_traffic/Conflict.cpp

```diff
@@ -135,38 +139,42 @@ double evaluate_spline(
       + coeffs[2] * t * t
       + coeffs[1] * t
       + coeffs[0]);
+  // double value = 0.0;
```

Comment:
If we're going to keep this in as a comment, let's add a remark about how the performance of this loop approach was profiled to be worse than the long version above.

Reply:
I think the comments have been removed in the latest version of the file.